web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx

```diff
@@ -58,7 +58,7 @@ const AnalysisRunnerSummary: React.FunctionComponent = () => {
     }
 
```

Comment:
Not in these changes, but FYI - if you have performance issues caused by `AnalysisRunnerGrid` re-rendering too many times, it might be caused by the `handleSort` function being re-created in a new memory address when this component is re-run. Do some profiling and wrap it in a `useCallback` if you need to.